Library/Formula/sslscan.rb

```diff
@@ -0,0 +1,23 @@
+class Sslscan < Formula
+  desc "Test SSL/TLS enabled services to discover supported cipher suites."
+  homepage "https://github.com/rbsec/sslscan"
+  url "https://github.com/rbsec/sslscan/archive/1.11.0-rbsec.tar.gz"
+  version "1.11.0"
+  sha256 "698dbf01b9af29aaddf6bce466f568762d852f7b71936861191a3b18d9dda6a5"
+  head "https://github.com/rbsec/sslscan.git"
+
+  bottle :unneeded
+
+  depends_on "openssl" => :build
+  depends_on "openssl"
+
+  def install
+    system "make"
```

Comment:
Presumably you shouldn't need both `make install` and `bin.install "sslscan"`. You'd also want to pass a prefix argument here if you intend to use `make`, since most things default to installing in `/usr` or `/usr/local`.